src/sidebar/components/test/notebook-result-count-test.js

```diff
@@ -58,17 +64,17 @@ describe('NotebookResultCount', () => {
       {
         thread: { children: [1] },
         visibleCount: 1,
-        expected: '1 thread (1 annotation)',
+        expected: '1 thread(1 annotation)',
```

Comment:
Visually the space is still there, so that is now assumed to come from the layout?

Reply:
Yeah. I admit this bugs me a bit, as it means that layout is affecting actual content. But the JSX reality here is also annoying (combinations of `{' '}`), whereas applying a single horizontal-rhythm rule via a mixin in SCSS is more "elegant." I hope I'm not making the wrong decision here. The two chunks of text content are in different heading elements, FWIW, so semantically there's still that.